### DISCUSSION TITLE
dB FS minimum value ?

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#727] [Q&A]
@ehpersonal38 
Hi !
Do you think there would be a way to keep only recordings for which BirdNet-Pi found a bird sound with dB FS value higher than (any number here) ?
I guess it may be really hard to create such an option, but I'm just asking... 😅
Thanks !

**Comment by ehpersonal38** [comment]
Two ideas:
Use `sox` in some kind of bash script to analyze the maximum/mean amplitude of the audio files in `~/BirdSongs/`, and delete it if the amplitude is lower than you want. https://stackoverflow.com/questions/43415353/explanation-of-audio-stat-using-sox

Or you could maybe just use the audio file size as a rough correlate to its overall loudness, since audio files with more data (louder) bird calls will inevitably be larger. I don't know how much larger though

**Reply by Fish31520** [reply]
@ehpersonal38
Thanks a lot !
Interesting, I didn't know louder sounds meant bigger files, that may be an interesting path to follow.

**Reply by ehpersonal38** [reply]
You're welcome!

Actually I just looked into this more, it seems the .mp3 files are all made to be the same size (48,192 bytes), BUT the file size of the associated spectrogram png files DO correlate with loudness as I described above ^. It's pretty rough though, and it's probably better to find some tool to actually analyze the waveforms
